sdk/synapse/synapse-artifacts/review/synapse-artifacts.api.md

```diff
@@ -0,0 +1,5227 @@
+## API Report File for "@azure/synapse-artifacts"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+import { HttpMethods } from '@azure/core-http';
+import { HttpOperationResponse } from '@azure/core-http';
+import { OperationArguments } from '@azure/core-http';
+import { OperationOptions } from '@azure/core-http';
+import { OperationSpec } from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PipelineOptions } from '@azure/core-http';
+import { Poller } from '@azure/core-lro';
+import { PollOperationState } from '@azure/core-lro';
+import { RestResponse } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+// @public
+export interface Activity {
+    [property: string]: any;
```

Comment:
putting an indexer on a type means it's effectively untyped as any property is valid. Can we nest these extended properties instead?

e.g. `additionalProperties: { [property: string]: any }`